src/content/1.7/modules/concepts/templating/import-js.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to import core js files 
+menuTitle: Import core js files
+weight: 4
+aliases:
+- 1.7/modules/templating/
```

Comment:
Where does this alias come from? I don't see any page there currently.

Reply:
src/content/1.7/modules/concepts/templating/admin-views.md 

Comes from this file, I removed it in import-js, do you want me to remove it from admin-views?

Reply:
Yep, that's a mistake, existing pages shouldn't be aliased.